internal/shim-sgx/src/handler/enarx.rs

```diff
@@ -12,18 +16,11 @@ impl<'a> EnarxSyscallHandler for super::Handler<'a> {
         &mut self,
         hash: UntrustedRef<'_, u8>,
         hash_len: libc::size_t,
-        _buf: UntrustedRefMut<'_, u8>,
-        _buf_len: libc::size_t,
+        buf: UntrustedRefMut<'_, u8>,
+        buf_len: libc::size_t,
     ) -> sallyport::Result {
-        self.trace("get_att", 0);
-
-        match hash.validate_slice(hash_len, self) {
-            None => Reply::from(Ok([SGX_QUOTE_SIZE.into(), SGX_TECH.into()])),
-            Some(..) => Reply::from(Err(libc::ENOSYS)),
-        }
-        .into()
```

Comment:
please don't remove the validation, even, if it currently does nothing.

Reply:
I cannot find a commit that would anything to validation between `mainline/main` and `origin/sgx_attestation_v2`.

Reply:
```
        let hash = match hash.validate_slice(hash_len, self) {
            None => {
                let rep: sallyport::Reply = Ok([SGX_QUOTE_SIZE.into(), SGX_TECH.into()]).into();
                return sallyport::Result::from(rep);
            }
            Some(h) => {
                if h.len() != 64 {
                    return Err(libc::EINVAL);
                }
                let mut hash = [0u8; 64];
                hash.copy_from_slice(h);
                hash
            }
        };
```
Doesn't this already take care of the validation?

Reply:
yes... I apologize... I shouldn't do reviews on the mobile after waking up.

Reply:
NP :-)